pkg/binder/framework/plugins/interpodaffinity/interpodaffinity.go

```diff
@@ -83,3 +93,29 @@ func New(_ runtime.Object, handle handle.BinderFrameworkHandle) (framework.Plugi
 		frameworkHandle: handle,
 	}, nil
 }
+
+func (pl *InterPodAffinity) updateStateByVictims(state *utils.PreFilterState, handle handle.BinderFrameworkHandle, victimsGroupByNode map[string]map[types.UID]*v1.Pod, targetPod *v1.Pod) error {
+	for nodeName, victimsMap := range victimsGroupByNode {
+		nodeInfo := handle.GetNodeInfo(nodeName)
+		if nodeInfo == nil {
+			continue
+		}
+
+		splitPlods, err := binderutils.SplitPods(victimsMap)
```

Comment:
typo: splitPlods -> launcherToPods. 
And an advice: SplitPods() -> GroupPodsByLauncher()

Reply:
Thanks! I have changed the names of both.